docs/specification.md

```diff
@@ -525,3 +625,16 @@ Thus, in a production setting, the full compatibility check can be reduced to a
 
 See: [ADR from 2023-07-12 on sorted name-length pairs](decision_record.md#2023-07-12-implementations-should-provide-sorted_name_length_pairs-and-comparison-endpoint)
 
+### F8. Adding new schema attributes
+
+A strength of this standard is that the schema definition can be modified for particular use cases, for example, by adding new attributes into a sequence collection.
+This will allow different communities to use the standard without necessarily needing to subscribe to identical schemas, allowing the standard to be more general useful.
```

Comment:
```suggestion
This will allow different communities to use the standard without necessarily needing to subscribe to identical schemas, allowing the standard to be more generally useful.
```